src/web/tools/Tracker.ts

```diff
@@ -0,0 +1,120 @@
+import { GHEvent } from './EventManager';
+interface TrackerElement {
+  lastX: number;
+  lastY: number;
+
+  timeStamp: number;
+
+  velocityX: number;
+  velocityY: number;
+}
+
+const VELOCITY_FACTOR = 0.2;
+export default class Tracker {
```

Comment:
```suggestion
export default class PointerTracker {
```

IMO this way it's more precise about what it's doing